### ISSUE TITLE
WiringPi Deprecated, what's the replacement?

### ISSUE THREAD

**Issue by GF-Huang** [sui77/rc-switch#384]
http://wiringpi.com/wiringpi-deprecated/

**Comment by keebenthusiast** [comment]
I made a version about a year ago which made use of the [pigpio](http://abyz.me.uk/rpi/pigpio/) library, if there is interest I could maybe make a pull request.

**Comment by redor303** [comment]
Yeah that would be really great. I tried it myself, but I think i messed something up with the interrupts.
